tailwind.config.ts

```diff
@@ -1,19 +1,103 @@
-import type { Config } from 'tailwindcss';
+/** @type {import('tailwindcss').Config} */
 
-const config: Config = {
+module.exports = {
+  darkMode: ['class'],
   content: [
-    './src/pages/**/*.{js,ts,jsx,tsx,mdx}',
-    './src/components/**/*.{js,ts,jsx,tsx,mdx}',
-    './src/app/**/*.{js,ts,jsx,tsx,mdx}',
+    './pages/**/*.{ts,tsx}',
+    './components/**/*.{ts,tsx}',
+    './app/**/*.{ts,tsx}',
+    './src/**/*.{ts,tsx}',
   ],
   theme: {
+    container: {
+      center: true,
+      padding: '2rem',
+      screens: {
+        '2xl': '1400px',
+      },
+      fontSize: {
+        sm: ['0.75rem', '1rem'],
+        md: ['1rem', '1.5rem'],
+        lg: ['1.125rem', '1.75rem'],
+      },
+    },
     extend: {
```

Comment:
I think it would be better to configure font families here, referring to the variables defined in the `fonts.ts`
```ts
extend: {
  fontFamily: {
    inter: ['var(--font-inter)', 'sans-serif'],
    outfit: ['var(--font-outfit)', 'sans-serif'],
    roboto: ['var(--font-roboto)', 'sans-serif'],
  },
```

Reply:
Moved to #42